src/CheckoutSdk/Payments/ResponseSourceConverter.cs

```diff
@@ -54,6 +54,10 @@ private static IResponseSource CreateRequest(string sourceType)
             {
                 case CardSource.TypeName:
                     return new CardSourceResponse();
+                case "ideal":
+                    return new AlternativePaymentSourceResponse();
+                case "giropay":
+                    return new AlternativePaymentSourceResponse();
                 default:
                     throw new NotImplementedException();
```

Comment:
@philippe-leonhardt-cko no need for the ideal/giropay cases, you can just make the default to return an `AlternativePaymentSourceResponse`